hack/verify-doctoc.sh

```diff
@@ -18,6 +18,8 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+command -v doctoc || echo "doctoc is not available on your system, skipping verification" && exit 0
+
```

Comment:
Doesn't this negate the purpose of this script?

Reply:
It can still be run locally, although I'm unaware if CI contains doctoc. If it doesn't this should probably move to a Dockerfile, wdyt?

Reply:
Running it locally doesn't help the use case that the verify script was intended for, though which is to ensure the TOCs are up to date on merge.

That said, I'm fine if we defer and track an issue to resolve later.

Reply:
Sounds good to me, this should have been caught in the other PR I admit